### DISCUSSION TITLE
G'day, I Really Need sum Help installing smtube-20.6.0-i686 on x32-bit Quickpup 21.04 Linux

### DISCUSSION THREAD

**Discussion by Max-Headroom88** [smplayer-dev/smtube#6] [General]
Targeted @ Old 2014 Acer C720 KromeBooks ( 1.4 GHz Dual Core intel Celerons w/ 2GB RAMemory ) of which I've got another Bakers Dozen of, Also Probably @ least 20 x 2011 Atom NetBooks I'm Determined 2 Rescue from Our Landfill. Naturally Mistfires x32-bit Quickpup 21.04 the ideal Puppy OS 4 such Legacy Hardware. However people these daze eXpect 2 be Able 2 View the *’UTube w/ their PC. So I Reckon this SMTube i686 should Fit the Criteria. eXcept that I've been having a Very Frustrating & Time Consuming Challenge Chasing Libs & Missing Dependencies! I’m just about Burnt Out!
The Latest Chapter of the Challenge Evidently iz related 2 incorrect installation of youtube-dl or perhaps wRong Version? SMTube Does Load But when I Clic’ on a Video I Receive...

SMTube failed to communicate with youtube-dl. Either it's not installed or it doesn't work correctly.
In order to play YouTube videos, SMTube needs an external application called youtube-dl.
SMTube can download and install this application for you. It will be downloaded from the official website and installed in /root/bin.

Would you like to proceeed?

I Clic’ Yes, But the Result iz a Window saying...
An error happened while downloading the file:
Error creating SSL context (error:140A90C4:SSL routines:func(169):reason(196))

which I don’t understand, of course I’ve already Searched & ReSearched Our Forum & the iNet, But Nuthing makes any scence.
Also Previously I Did Try the QT5 Version of SMTube, But have been Advized by Other Puppy Developers & Forum Members that it’ll Bring Big Problems, I can’t aRgue w/ their eXperience & Advice after all of the Chasing Libs & Missing Dependencies I’ve since eNdured! However I still wunder if there iz a Good Valid Reason 2 Persist w/ Qt5 & I was just doing sumthing wRong?!?

Edit. after Yet another Clean install I'm Missing libqtwebkit & openldap Dependencies & from Utilities Dynamic Dependencies these...
libpng15.so.15 => not found
libwebp.so.4 => not found
libgstapp-0.10.so.0 => not found
libgstinterfaces-0.10.so.0 => not found
libgstpbutils-0.10.so.0 => not found
libgstvideo-0.10.so.0 => not found
libgstbase-0.10.so.0 => not found
libgstreamer-0.10.so.0 => not found
libpng15.so.15 (PNG15_0) => not found

Again Nun of these aRe in the Puppy Package Manager after Updating it.

Please Help...

Please & Thanx!

;-)K

Ps. Please don't direct me 2 the Puppy Linux Forum because I've been there done that, they're Mostly Responsive, But Sadly in this instance not, so I've got no choice but 2 cast the net a lil wider...

**Comment by smplayer-dev** [comment]
Maybe the openssl library is not installed in your system and smtube fails to open https URLs. Try installing it.
